### PR TITLE
Add README.md, Code of Conduct and LICENSE

### PR DESCRIPTION
**Pull request by bhavin192**
- Add CC BY-SA 4.0 license to the repository
  - https://creativecommons.org/licenses/by-sa/4.0/
- Add Code of Conduct for the PythonPune meetup group
  - Adopted from Pyladies CoC https://www.pyladies.com/CodeOfConduct/
- Update README.md with more details about meetup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+
+* [Submit a talk](#submit-a-talkworkshop)
+* [Monthly newsletter](#monthly-newsletter)
+* [Summary of past events](#summary-of-past-events)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Submit a talk/workshop
+If you want to submit a talk for upcoming events, then you can create
+a new issue on this repository. Just fill in the requested details and
```
Comment (akshay196) [inline README.md]: How about we hyper-link the text `create a new issue` with URL https://github.com/pythonpune/meetup-talks/issues/new/choose?

**Comment by akshay196** [comment]
Other looks good to me. Thanks @bhavin192 for this PR.

File: README.md (hunk 1 above)
Comment (bhavin192) [reply README.md]: Right, I forgot to do that.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,60 @@
...
+
+## Code of Conduct
+Code of Conduct for [PythonPune meetup
+group](https://meetup.com/PythonPune) can be found
+[here](./CODE_OF_CONDUCT.md).
+
+## License
+This repository is licensed under [Creative Commons
+Attribution-ShareAlike 4.0 International (CC BY-SA
+4.0)](https://creativecommons.org/licenses/by-sa/4.0/). The slides,
```
Comment (raukadah) [inline README.md]: Here it is 4.0 but above https://github.com/pythonpune/meetup-talks/pull/39/files#diff-a1ee87dafebc22cbd96979f1b2b7e837R41 it is mentioned 3.0, Any reason.

Comment (bhavin192) [reply README.md]: It's the license of COC. As mentioned further in the License section "The slides, code snippets or any other images in this repository might have a different license as well as copyright, please check before reusing the material."

Comment (raukadah) [reply README.md]: Ack. thanks!

**Pull request merged**